test/sql/storage/compression/dict_fsst/test_dict_fsst_with_smaller_block_size.test

```diff
@@ -0,0 +1,12 @@
+# name: test/sql/storage/compression/dict_fsst/test_dict_fsst_with_smaller_block_size.test
+# description: Test storage with dictionary compression and a smaller block size.
+# group: [dict_fsst]
+
+statement ok
+SET storage_compatibility_version='latest';
+
+statement ok
+ATTACH '__TEST_DIR__/partial_manager.db' AS db (BLOCK_SIZE 16384);
+
+statement ok
```

Comment:
Does this contain nulls?
Don't we need a "restart" to verify correctness on reload?

This test also doesnt have a `load` so it's not actually testing any storage

Reply:
Also, doesn't it only auto-checkpoint if the WAL threshold is reached?

Reply:
> Can we verify that the created table contains nulls and non-nulls?
> Also scan the table, as that might otherwise hide a bug that is only triggered when the segments are read

Good call, will do 👍 

> Also, doesn't it only auto-checkpoint if the WAL threshold is reached?

Hmm, it was checkpointing on my breakpoint, I didn't dive to deep into the why. 😅 But yes, it auto-checkpoints, if the WAL threshold is exceeded. Anyways, I'll just add a `DETACH` and `reATTACH` to be sure.